commit 38d248aadd9d8c7ad017ba70b88c5dfe6ec15e83
Author: Benjamin Kramer <benny.kra@googlemail.com>
Date:   Thu Mar 24 10:55:02 2022 +0100

    [bazel] Set the default of LLVM_UNREACHABLE_OPTIMIZE to 1
    
    This matches the CMake default. LLVM_UNREACHABLE_OPTIMIZE=0 doesn't
    build with gcc

diff --git a/utils/bazel/llvm-project-overlay/llvm/include/llvm/Config/llvm-config.h b/utils/bazel/llvm-project-overlay/llvm/include/llvm/Config/llvm-config.h
index e83d6e94c581..59d8aa6c0c6d 100644
--- a/utils/bazel/llvm-project-overlay/llvm/include/llvm/Config/llvm-config.h
+++ b/utils/bazel/llvm-project-overlay/llvm/include/llvm/Config/llvm-config.h
@@ -122,6 +122,6 @@
 
 /* Define if llvm_unreachable should be optimized with undefined behavior
  * in non assert builds */
-/* #undef LLVM_UNREACHABLE_OPTIMIZE */
+#define LLVM_UNREACHABLE_OPTIMIZE 1
 
 #endif
